std/typecons.d

```diff
@@ -6773,16 +6782,23 @@ pure @system unittest
     import std.conv : to;
     // Check that string conversion is transparent for refcounted
     // structs that do not have either toString or alias this.
-    struct A { Object a; }
+    static struct A { Object a; }
     auto a  = A(new Object());
     auto r = refCounted(a);
     assert(to!string(r) == to!string(a));
     assert(to!string(cast(const) r) == to!string(cast(const) a));
     // Check that string conversion is still transparent for refcounted
     // structs that have alias this.
-    struct B { int b; alias b this; }
-    struct C { B b; alias b this; }
+    static struct B { int b; alias b this; }
+    static struct C { B b; alias b this; }
     assert(to!string(refCounted(C(B(123)))) == to!string(C(B(123))));
+    // https://issues.dlang.org/show_bug.cgi?id=22093
+    // Check that uninitialized refcounted structs that previously could be
```

Comment:
Technically it should be "default-initialized", not uninitialized.

Reply:
I could have phrased that better. I meant "not initialized" to be understood as `RefCounted.refCountedStore.isInitialized == false`.